Doc/whatsnew/3.13.rst

```diff
@@ -1697,6 +1697,17 @@ New Features
   more information.
   (Contributed by Victor Stinner in :gh:`111696`.)
 
+* Add :c:func:`Py_GetConstant` and :c:func:`Py_GetConstantBorrowed` functions
+  to get constants. For example, ``Py_GetConstant(Py_ZERO_IDX)`` returns a
+  :term:`strong reference` to the constant zero.
+  (Contributed by Victor Stinner in :gh:`115754`.)
+
+* In the limited C API version 3.13, getting ``Py_None``, ``Py_False``,
+  ``Py_True``, ``Py_Ellipsis`` and ``Py_NotImplemented`` singletons is now
+  implemented as function calls at the stable ABI level to hide implementation
+  details. Getting these constants still return borrowed references.
+  (Contributed by Victor Stinner in :gh:`115754`.)
+
```

Comment:
I don't think this is a user-visible change.

Reply:
I removed it from What's New but I prefer to keep it in the Changelog since it changes the ABI and has a negative impact on performance. At least, users can see the relationship between this changes and effects that they see.